c/zss.c

```diff
@@ -727,6 +727,43 @@ static int checkLoggingVerbosity(const char *serverConfigFile, char *pluginIdent
   return ZOWE_LOG_INFO;
 }
 
+static void checkAndSetServiceLoglevel(const char *serverConfigFile, const char *pluginIdentifier, 
```

Comment:
Is this really `pluginIdentifier`? It's a dataservice identifier.

Reply:
renamed parameter as dataserviceIdentifier